src/sagemaker_training/mpi.py

```diff
@@ -23,11 +23,29 @@
 import psutil
 
 import gethostname
-from sagemaker_training import logging_config, process, timeout
+from sagemaker_training import environment, errors, logging_config, process, timeout
 
 logger = logging_config.get_logger()
 logging.getLogger("paramiko").setLevel(logging.INFO)
 
+try:
+    from smdistributed.modelparallel.backend import exceptions
+
+    # list of exceptions SMMP wants training toolkit to catch and log
+    exception_classes = [x for x in dir(exceptions) if isclass(getattr(exceptions, x))]
+except ImportError:
+    logger.info("No exception classes found in smdistributed.modelparallel")
+    exception_classes = []
+try:
```

Comment:
can we leave an empty line for readability reasons between multiple `try` `except` blocks?